docs/compilers/Visual Basic/Single-line if with End as statement.md

```diff
@@ -0,0 +1,25 @@
+# Single-line if with `End` keyword as statement
+
+As reported in [#45158](https://github.com/dotnet/roslyn/issues/45158), it was possible to use something like the following until Visual Studio version (TODO: UNKNOWN):
```

Comment:
I've only tried the snippet on Visual Studio 2015 and Visual Studio 2019. So, I don't know if this breaking change was introduced in 2017 or 2019.